modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/index.ts

```diff
@@ -16,88 +16,178 @@
 
+// @ts-ignore
+
 export {default as BREAKPOINTS} from './liferay/breakpoints';
 
+// @ts-ignore
+
 export {default as CompatibilityEventProxy} from './liferay/CompatibilityEventProxy.es';
 
+// @ts-ignore
+
 export {default as DefaultEventHandler} from './liferay/DefaultEventHandler.es';
+
+// @ts-ignore
+
 export {default as Disposable} from './liferay/events/Disposable';
+
+// @ts-ignore
+
 export {default as EventEmitter} from './liferay/events/EventEmitter';
+
+// @ts-ignore
+
 export {default as EventHandler} from './liferay/events/EventHandler';
+
+// @ts-ignore
+
 export {default as ItemSelectorDialog} from './liferay/ItemSelectorDialog.es';
+
+// @ts-ignore
+
 export {default as PortletBase} from './liferay/PortletBase.es';
 
 // Modal API
 
+// @ts-ignore
+
 export {openModal, openSelectionModal} from './liferay/modal/Modal';
 
+// @ts-ignore
+
 export {default as openSimpleInputModal} from './liferay/modal/commands/OpenSimpleInputModal.es';
 
 // PortletURL API
 
+// @ts-ignore
+
 export {default as createActionURL} from './liferay/util/portlet_url/create_action_url.es';
 
+// @ts-ignore
+
 export {default as createPortletURL} from './liferay/util/portlet_url/create_portlet_url.es';
 
+// @ts-ignore
+
 export {default as createRenderURL} from './liferay/util/portlet_url/create_render_url.es';
 
+// @ts-ignore
+
 export {default as createResourceURL} from './liferay/util/portlet_url/create_resource_url.es';
 
 // Align API
 
-export {
-	ALIGN_POSITIONS,
-	align,
-	getAlignBestRegion,
-	getAlignRegion,
-	suggestAlignBestRegion,
-} from './liferay/align';
+// @ts-ignore
+
+export {ALIGN_POSITIONS, align, getAlignBestRegion} from './liferay/align';
+
+// @ts-ignore
+
+export {getAlignRegion, suggestAlignBestRegion} from './liferay/align';
 
 // Session API
 
+// @ts-ignore
+
 export {getSessionValue, setSessionValue} from './liferay/util/session.es';
 
 // Toast API
 
+// @ts-ignore
+
 export {openToast} from './liferay/toast/commands/OpenToast.es';
 
 // Throttle API
 
+// @ts-ignore
+
 export {default as throttle} from './liferay/throttle.es';
 
 // Util API
 
+// @ts-ignore
+
 export {default as addParams} from './liferay/util/add_params';
+
+// @ts-ignore
+
 export {default as buildFragment} from './liferay/util/build_fragment';
+
+// @ts-ignore
+
 export {default as fetch} from './liferay/util/fetch.es';
+
+// @ts-ignore
+
 export {default as focusFormField} from './liferay/util/focus_form_field';
+
+// @ts-ignore
+
 export {default as getPortletId} from './liferay/util/get_portlet_id';
+
+// @ts-ignore
+
 export {default as inBrowserView} from './liferay/util/in_browser_view';
+
+// @ts-ignore
+
 export {default as isObject} from './liferay/util/is_object';
+
+// @ts-ignore
+
 export {default as isPhone} from './liferay/util/is_phone';
+
+// @ts-ignore
+
 export {default as isTablet} from './liferay/util/is_tablet';
+
+// @ts-ignore
+
 export {default as navigate} from './liferay/util/navigate.es';
+
+// @ts-ignore
+
```

Comment:
Is there a way to `@ts-ignore` the whole file?

Reply:
Yes, you can do `// @ts-nocheck`, but then I wouldn't be able to slowly clean them up 😉